src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -393,7 +393,10 @@ static int bus_socket_auth_verify_server(sd_bus *b) {
                                 r = bus_socket_auth_write(b, "REJECTED\r\n");
                         else {
                                 b->auth = BUS_AUTH_ANONYMOUS;
-                                r = bus_socket_auth_write_ok(b);
+                                if (l <= 14)
```

Comment:
hmm, can we change this to not require the hardcoding of "14" here? I mean, we hardcode it twice already, and I'd rather have fewer than more of that. Hence, I'd propose to define an enum:

```
enum {
        BUS_SOCKET_AUTH_REJECT,
        BUS_SOCKET_AUTH_NO_DATA,
        BUS_SOCKET_AUTH_WITH_DATA,
};
```

And then make verify_anonymous_token() return those values. After all it checks whether DATA is specified anyway, internally, and hence can just let us know directly.

Reply:
Shouldn't the original author of these magic numbers fix it? Life is so unfair!!!111

Reply:
well, those existing magic nrs would remain, but we wouldn't copy them a 3rd time. (unless of course you also want to remove the first two, which I'd of course very much appreciated...)